### PR TITLE
Set the seed directly on Random

### PR DESCRIPTION
**Pull request by owsi**
DoubleMatrix/rand seed merely creates a vector of length seed. it is a
no-op since its return is not used or stored.

The comment is still valid as it is indeed, global.

### CODE REVIEW

**Comment by whilo** [comment]
Thank you again. I introduced it, but obviously made a stupid mistake here. Do you get reproducible runs now?

**Comment by owsi** [comment]
appears so, I haven't focused on that, just giving a quick lesson on RBM's to some co-workers

**Comment by whilo** [comment]
Ah, nice. Do you have a background with Boltzmann machines? I am working with them on spiking neurons for research and use them for pretraining of deep nets.

I have done some heavy manual inlining in `jblas.clj` (`outer-product` and `cond-prob-batch`) adviced by jvisualvm, but I am very unhappy with it. Actually `core.matrix` should be barely slower, if you like to you could have a look at using `core.matrix` there again. I normally use `theano` in Python on the GPU, which was even 4x times faster on my laptop on the CPU. With the GPU it is at least 10x times faster. So atm. I am working on a `core.matrix` implementation of `nd4j` (which has a GPU backend) to get access to the deeplearning4j code (also for finetuning with backprop, CNNs, etc.): https://github.com/whilo/clj-nd4j
